src/app/widgets/Visualizer/index.jsx

```diff
@@ -1277,8 +1294,47 @@ class VisualizerWidget extends PureComponent {
                 controller.command('macro:run', macroID, controller.context);
             }
         },
-        VISUALIZER_VIEW_CYCLE: {
+        JOG_A_PLUS: { // Jog A+
+            id: 71,
+            title: 'Jog: A+',
+            keys: ['ctrl', '6'].join('+'),
+            cmd: 'JOG',
+            payload: {
+                axis: { a: 1 },
+            },
+            preventDefault: false,
+            isActive: true,
+            category: JOGGING_CATEGORY,
+            callback: this.rotaryAxisFunctions.JOG_PLUS
+        },
+        JOG_A_MINUS: { // Jog A-
             id: 72,
+            title: 'Jog: A-',
+            keys: ['ctrl', '4'].join('+'),
+            cmd: 'JOG',
+            payload: {
+                axis: { a: -1 },
+            },
+            preventDefault: false,
+            isActive: true,
+            category: JOGGING_CATEGORY,
+            callback: this.rotaryAxisFunctions.JOG_MINUS
+        },
+        ROTARY_AXIS: { // Rotary Axis ON/OFF
+            id: 73,
+            title: 'Rotary Axis',
+            keys: ['ctrl', '5'].join('+'),
+            cmd: 'ROTARY_AXIS',
+            payload: {
+                command: 'rotaryAxis:updateState',
+            },
+            preventDefault: false,
+            isActive: true,
+            category: GENERAL_CATEGORY,
+            callback: this.rotaryAxisFunctions.UPDATE_STATUS
+        },
```

Comment:
Not necessary to have add shortcuts here anymore as the useKeybinding hook only takes in values from the shuttleControlEvents